extensions/amazon-dynamodb/runtime/src/main/java/io/quarkus/dynamodb/runtime/ApacheHttpClientConfig.java

```diff
@@ -0,0 +1,149 @@
+package io.quarkus.dynamodb.runtime;
+
+import java.net.URI;
+import java.time.Duration;
+import java.util.List;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class ApacheHttpClientConfig {
+
+    /**
+     * The amount of time to wait when acquiring a connection from the pool before giving up and timing out.
+     *
+     * Default is 10 seconds.
```

Comment:
I wonder if we should make them hard defaults so that they are properly documented. WDYT?

Reply:
Make sense, but I'm wondering now if I should document it in here. It's basically because those default are buried deep in the SDK builders and took me some time to find them out. So I think I should rather remove them. WDYT ?